draft-poidt-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -417,7 +417,10 @@ The use cases a generally applicable for both the traditional packet over optica
 
 #### Coherent DWDM pluggable insertion in the router linecard port ('valid for coherent pluggable')
-Once a pluggable module is inserted in the proper linecard port, the host device must recognise the hardware component (ZR+ pluggable module) and expose its attributes and capabilities to the controller. For example, ZR+ modules can share the operational-mode that summarize the most important pluggable characteristics (such as FEC type, modulation format, baud rate, bit rate, etc.). If the hardware component has been successfully recognised, the host device is then ready to create and expose the necessary logical arrangements.
+Once a pluggable module is inserted in the proper linecard port, the host device must recognise the hardware component (e.g. 400G ZR+ pluggable module) and expose its attributes and capabilities to the controller. For example, ZR+ modules can share the operational-mode-IDs supported that summarize the most important pluggable characteristics (such as FEC type, modulation format, baud rate, bit rate, etc.). If the hardware component has been successfully recognised, the host device is then ready to create and expose the necessary logical arrangements.
+
+Several coherent pluggables seem to come with a factory default set of provisioning parameters (e.g. default channel number, default launched power, default application code id, laser-on, admin-state enabled etc.). This factory default set of provisioning parameters varies from manufacturer to manufacturer. This can allow a “plug&play” mode of operation over point-to-point connections (e.g. single wavelength over dark fiber) of course limited to bookended solutions from the same Vendor and same factory default provisioning parameters. However, when the optical connection between two pluggables is targeted to run over a DWDM Open Line System (OLS) network, optical validation & planning step is first required to determine the right target provisioning parameters values to be set in the pluggables before interconnecting them to their respective ROADM to avoid to impact any other existing optical channels already up and running in the OLS network.
+It is critical for operators to have the same kind of commissioning phase independently of the deployment scenario: point-to-point vs ROADM meshed OLS network. As a consequence, the use of factory default provisioning parameters may be fine but they shall always be able to be overwritten through router CLI or through Packet PNC to another set of  default provisioning parameters defined by the operator that will change from pluggable to pluggable when deployed over an OLS network. A reset of the coherent pluggable (through router CLI or through Packet PNC or due to a power off/on) shall always go back to this operator’s default set of provisioning parameters where, for example, the laser-state shall be ‘Off’ and admin-state ‘disabled’.
```

Comment:
I'm not sure I get the idea behind "the same kind of commissioning phase". The following sentences allow different approaches (which is fine), provided the operator remains in control in all cases: that's what we should promote, but the initial wording is confusing.
I would also reserve the term "default" to refer to the factory values and make it clearer operators want to control the running parameters, even on startup, whatever the factory values.
NEW: "the use of factory default provisioning parameters may be fine but the operator must always be able to override them through router CLI or Packet PNC to another set of startup provisioning parameters that will change from pluggable to pluggable when deployed over an OLS network. A reset of the pluggable (through router CLI/Packet PNC, or due to a power off/on) shall always go back to this operator-custom set of provisioning parameters (e.g., laser-state to ‘Off’ and admin-state to ‘disabled’)."